packages/web-component/src/lib/descope-wc/BaseDescopeWc.ts

```diff
@@ -485,6 +485,15 @@ class BaseDescopeWc extends HTMLElement {
       // in case there is a single "generic" button on the page, click on it
       if (genericButtons.length === 1) {
         genericButtons[0].click();
+      } else if (genericButtons.length === 0) {
+        const ssoButtons = Array.from(buttons).filter(
```

Comment:
We can use `find` instead of filter and get the button or undefined. This will make the `length === 1` unnecessary, and will always return the first button, in case there are multiple SSO buttons (if that case is possible).

Reply:
See few lines above, it does the exact same so I used the same paradigm, if we change we should change other places as well, and not sure I want to do that now, since it might be risky change.

Reply:
Just checked and `find` is not good here, since we want to not click anything if there are more than one button (and not click the first one).